enum.h

```diff
@@ -37,7 +61,7 @@
 #   endif
 #endif
 
-#ifdef _MSC_VER
+#ifdef _MSVC_VER
```

Comment:
Should be `_MSC_VER`, not `_MSVC_VER`, i.e. there should be no diff on this line.

Reply:
crap!

just fixed